sdk/typescript/runtime/template/package.json

```diff
@@ -1,6 +1,7 @@
 {
+  "type": "module",
```

Comment:
I am assuming that this change is related to `npm pkg set type=module`. Wondering the same thing here.

Reply:
Same reason, for new init :D